lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -3524,7 +3700,7 @@ namespace Js
         newFunctionBody->AllocateLiteralRegexArray();
         for(uint i = 0; i < this->literalRegexCount; ++i)
         {
-            const auto literalRegex = this->literalRegexes[i];
+            const auto literalRegex = this->GetLiteralRegexs()[i];
```

Comment:
Spelling of method, should probably be `GetLiteralRegexes` to match the field name.